_pages/en/mandate.md

```diff
@@ -16,43 +16,43 @@ The [Canadian Experience](https://github.com/sara-sabr/ITStrategy/blob/master/wi
 
 ## Mission (team's reason for existence)
 
-Enable the strategic value of IT within ESDC by reducing its risks to accelerate business flexibility.
+Enable the strategic value of IT within ESDC to increase business flexibility.
```

Comment:
I think it's good to remove "by reducing risks", since that's how we're going to do it and that not the mission.

I would change flexibility to agility since that the term we use in the medium and long term pictures

Reply:
Agreed about agility

Reply:
The reducing risks is important I would think

Reply:
risk of IT?  there are f few other places in that mention risk and reducing risk, I'm thinking we should remove some focus on lead time for a change or a thing, time to recover, and change/thing failure rate.